AUTHORS.md

```diff
@@ -152,3 +152,4 @@ Ville Skyttä
 Vu Nhat Chuong
 Wasif Shahzad
 Youhei Sakurai
+Rajvardhan
```

Comment:
Please ensure the name placement is sorted alphabetically (Maybe do this after you've merged in the new changes since we've recently switched from `AUTHORS.md` to `CONTRIBUTORS` recently)